pylint/exceptions.py

```diff
@@ -0,0 +1,14 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
No idea. :-) I only see copyright headers on a few files in the whole repo - are you sure this is needed?